src/messages/messages.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please use "..." for all the messages starts with a "xxxing" verb and please use "..." with consistent style (either use a space in front of it or no space)